JAGPropertyConverterTests/JAGPropertyConverterTest.m

```diff
@@ -132,6 +142,24 @@ - (void) testToModelWithDifferntPropertyName {
     STAssertEqualObjects(testModel.differentNameProperty, @"new name", @"");
 }
 
+- (void)testToModelWithKeypathCustomMapping {
+    // custom mapping is directly implemented by TestModel with <JAGPropertyMappingProtocol>
+    NSDictionary *dict = [NSDictionary dictionaryWithObjectsAndKeys:
+                          @"M123122", @"testModelID",
+                          [NSNumber numberWithInt:5], @"intProperty",
+                          [NSNumber numberWithBool:YES], @"boolProperty",
+                          @"new name" , @"someProperty",
+                          @"MPID001", @"keypathProperty1",
+                          @"MPID002", @"keypathProperty2",
```

Comment:
here also use direct keypath.

in fact it would be better to create 2 separate tests. one testing keypath directly (using keypath as key here in dict) and a second one using custom mapping, as you have done here.

hmm i think it would better put all new keypath tests on the bottom of the file